Cargo.toml

```diff
@@ -485,6 +485,7 @@ transforms-logs = [
 ]
 transforms-metrics = [
   "transforms-add_tags",
+  "transforms-aggregate",
```

Comment:
Absolutely. Some of the practices have changed since it was written last year, and the team has grown as well, meaning more and different reviewers. My comment on naming was more reflective of our sinks, which increasingly use distinguishing names. As you rightly point out, the `reduce` transform does not specifically indicate it operates on logs, nor do most of the transforms, so this naming is probably fine.